airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -409,16 +409,22 @@ def supports_external_link(self) -> bool:
         return bool(self.frontend)
 
 
-def safe_attrgetter(*items, obj, default):
+def getattr_nested(obj, item, default):
     """
-    Get items from obj but return default if not found
+    Get item from obj but return default if not found
+
+    E.g. calling ``getattr_nested('b.c', a, "NA")`` will return
+    ``a.b.c`` if such a value exists
 
     :meta private:
     """
-    val = None
+    NOTSET = object()
+    val = NOTSET
     try:
-        val = attrgetter(*items)(obj)
+        val = attrgetter(item)(obj)
     except AttributeError:
         pass
```

Comment:
```python
try:
    return attrgetter(item)(obj)
except AttributeError:
    return default
```

I think this would work…?

Reply:
yes, now that we're  no longer being as "helpful" thanks... updated